CMakePresets.json

```diff
@@ -271,6 +271,17 @@
         "CCCL_ENABLE_EXAMPLES": true,
         "CCCL_ENABLE_TESTING": true
       }
+    },
+    {
+      "name": "all-dev-debug",
+      "displayName": "all-dev debug",
+      "inherits": "all-dev",
+      "cacheVariables":{
+        "CCCL_ENABLE_BENCHMARKS": false,
+        "CMAKE_BUILD_TYPE": "Debug",
+        "CMAKE_CUDA_FLAGS": "-G",
+        "CMAKE_CUDA_ARCHITECTURES": "native"
```

Comment:
question: does 'native' work for us?

Reply:
What do you mean us?

Reply:
> What do you mean us?

If I set "native" as `CMAKE_CUDA_ARCHITECTURES`, will CCCL project be properly configured and built?

Reply:
I recently did update libcu++ to be able to build it against native AFAIK

Reply:
we still got cub and thrust tho, so let's keep it hardcoded?